catalogs/catalog-hadoop/build.gradle.kts

```diff
@@ -91,6 +97,12 @@ tasks {
     from("build/libs") {
       exclude("slf4j-*.jar")
       exclude("guava-*.jar")
+      exclude("curator-*.jar")
+      exclude("netty-*.jar")
+      exclude("snappy-*.jar")
+      exclude("zookeeper-*.jar")
+      exclude("jetty-*.jar")
+      exclude("javax.servlet-*.jar")
```

Comment:
Looks like some jetty libraries still exist in your screenshot, can you please check again.

Reply:
done.
![image](https://github.com/user-attachments/assets/ad45fca1-fbe2-428f-a521-1d0fa1b0d509)